widget/src/components/buttons/Danger.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'preact';
+import classNames from 'classnames';
+
+const Danger = (props: React.ComponentProps<'button'>) =>
```

Comment:
To avoid the type cast below:
```suggestion
const Danger = (props: React.ComponentProps<'button'> & {className: string}) =>
```